openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -126,28 +131,81 @@ class TestCourseHomePageAccess(CourseHomePageTestCase):
     """
     Test access to the course home page.
     """
+
+    def setUp(self):
+        super(TestCourseHomePageAccess, self).setUp()
+
+        # Make this a verified course so that an upgrade message might be shown
+        add_course_mode(self.course, upgrade_deadline_expired=False)
+
+        # Add a welcome message
+        create_course_update(self.course, self.staff_user, TEST_WELCOME_MESSAGE)
+
+    def _create_test_user(self, user_type):
+        """
+        Create a test user of the specified type.
+
+        Valid values for user type:
+          anonymous - an anonymous user
```

Comment:
Enum is only supported in Python 3, as best I can tell:

https://stackoverflow.com/questions/36932/how-can-i-represent-an-enum-in-python

I did a search through the codebase and couldn't find any enums. Do you have an idiom in mind?